src/faultlog/faultlog_main.cpp

```diff
@@ -27,6 +33,36 @@ using Severity = sdbusplus::xyz::openbmc_project::Logging::server::Entry::Level;
 using Binary = std::vector<uint8_t>;
 using PropVariant = sdbusplus::utility::dedup_variant_t<Binary>;
 
+/**
+ * @brief To init phal library for use power system specific device tree
+ *
+ * @return void
+ */
+void initPHAL()
+{
+    // Set PDBG_DTB environment variable to use interested phal cec device tree
+    if (setenv("PDBG_DTB", PHAL_DEVTREE, 1))
```

Comment:
Would it be a better idea to move this is to pdbg_targets_init ? May be we could set a default value to pdbg_dtb if not set. 
Just a thought!!!

Reply:
it might break lot of places like prd, where they are setting there own custom dtb file and calling pdbg_targets_init expecting it to consume.